hieradata/class/email_alert_api.yaml

```diff
@@ -0,0 +1,5 @@
+---
+
+govuk::node::s_base::apps:
```

Comment:
We shouldn't add apps in this way anymore. Add it in `common.yaml` in both hieradata and hieradata_aws in [this block](https://github.com/alphagov/govuk-puppet/blob/master/hieradata/common.yaml#L10-L122)

Reply:
Fixed